test/modularize_post_js.js

```diff
@@ -0,0 +1,10 @@
+#preprocess
+
+// This file get included via `--extern-post-js` in order to run tests
+// that are built with `-sMODULARIZE`.
+
+#if PTHREADS
+// Avoid instantiating the module on pthreads.
+if (!isPthread)
```

Comment:
`isPthread` is only defined inside the emscripten JS output, isn't it? How is it usable from outside it, in `--extern-post-js`?

(The code this replaces does things like check `typeof importScripts != "function"` which will work anywhere.)

Reply:
Actually `isPthread` is specifically available outside of the module, when MODULARIZE is used.  See https://github.com/emscripten-core/emscripten/blob/3d58456a7d37a8fb269295189ed00c554cb3f26d/tools/link.py#L2446-L2460
 
Inside the module we use `ENVIRONMENT_IS_PTHREAD`.

Reply:
Thanks, I wasn't aware we added that to the outside scope, interesting.